specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -0,0 +1,2623 @@
+            "Failed",
+            "Complete"
+          ],
+          "x-ms-enum": {
+            "name": "NamespaceHydrationStatusType",
+            "modelAsString": true
+          }
+        },
+        "progress" : {
+          "description" : "Progress information for the namespace hydration",
+          "type" : "object",
+          "properties" : {
+            "blobCount" : {
+              "description" : "The number of Blobs processed.",
+              "type": "integer"
+            },
+            "size" : {
+              "type" : "integer",
+              "description" : "Aggregate size of blobs ingested so far"
+            },
+            "rate" : {
+              "type" : "integer",
+              "description": "Rate of ingest accounting for the last 30 seconds"
+            },
+            "blobName" : {
+              "type" : "string",
+              "description" : "Name of the current blob"
+            },
+            "errorCount" : {
+              "type" : "integer",
+              "description" :"Number of failures"
+            }
+          }
+        }
+      }
+    },
+    "amlFilesystemArchiveInfo": {
+      "type":"object",
+      "description": "Information required to execute the archive operation",
+      "properties": {
+        "filesystemPath" : {
+          "type" : "string",
+          "description" : "The path to the filesystem to archive."
+        }
+      }
+    },
+    "amlFilesystemArchive" : {
+      "description" : "Information about the amlFilesystem archive",
+      "type" : "object",
+      "readOnly": true,
+      "properties": {
+        "filesystemPath" : {
+          "readOnly": true,
+          "type" : "string",
+          "description" : "The path to the filesystem to archive."
+        },
+        "status" : {
+          "description" : "The status of the archive",
+          "type" : "object",
+          "readOnly" : true,
+          "properties": {
+            "state" : {
+              "readOnly": true,
+              "description" : "The state of the archive operation",
+              "type" : "string",
+              "enum" : [
+                "Idle",
+                "InProgress"
+              ],
+              "x-ms-enum": {
+                "name": "ArchiveStatusType",
+                "modelAsString": true
+              }
+            },
+            "lastCompletionTime" : {
+              "readOnly" : true,
+              "description" : "The time of the last completed archive operation",
+              "type" : "string",
+              "format" : "date-time"
+            },
+            "lastStartedTime" : {
+              "readOnly" : true,
+              "description" : "The time the latest archive operation started",
+              "type" : "string",
+              "format" : "date-time"
+            },
+            "percentComplete" : {
+              "readOnly" : true,
+              "description" : "The completion percentage of the archive",
```

Comment:
Do we know a percent complete? To answer that, we need to know how much data we have not archived. (This may be a question for BrianB)

Reply:
I believe we should be able to compute this since Robinhood has ways for us to query the amount of remaining dirty data and/or files to archive. Unfortunately, I need to be able to build a cluster with lots of dirty data and copytools running to decide how we will best do this in our environment.  It's possible that we'll need to change this field to a number of bytes at a later date, but for a first round review percentageComplete is probably good enough as long as we can be open to changing it later before final submission, if needed.